book/integration-service.md

```diff
@@ -137,6 +137,11 @@ Following the [annotation guidelines](https://docs.google.com/document/d/1gyXM3p
 ```
 The Integration service will copy the annotations and labels from the Build PipelineRun and append those to the Test PipelineRuns for traceability across the system per [Labels and Annotations for StoneSoup pipelines](https://docs.google.com/document/d/1fJq4LDakLfcAPvOOoxxZNWJ_cuQ1ew9jfBjWa-fEGLE/edit#) and [StoneSoup builds and tests PRs](https://docs.google.com/document/d/113XTplEWRM63aIzk7WwgLruUBu2O7xVy-Zd_U6yjYr0/edit#)
 
+The Label below provides users an option whether the result of a pipelineRun created according to the IntegrationTestScenario will be taken into account when determining if the Snapshot has passed all required testing. It will be copied to the subsequent Test PipelineRuns. 
```

Comment:
I think that I am still confused by this sentence. Maybe wording it something like this will be clearer (and removing the use of option + optional)?

> The `test.appstudio.openshift.io/optional` label can leveraged by the user to affect the determination of whether a Snapshot has passed all required testing. If it is set to `true`, then the result of the PipelineRun created according to the IntegrationTestScenario will need to be passing for the overall Snapshot to pass all tests.

Don't take the words verbatim as I think they are still not accurate, just as a suggestion for how I think we can get clarity.

Reply:
If the label is set to "true", the result of the PipelineRun will not be taken into account when determining if the Snapshot has passed all required testing.

Reply:
How about the updated description?

Reply:
That works, thanks. Can you just comment on the default behavior if the label is absent? Or will this label never be absent in the future because some service is always setting it to the default of `false`?

Reply:
It's descibed - If the label is not defined in an IntegrationTestScenario, integration service will consider it as "false".

Reply:
Many thanks. Sorry for the oversight. Maybe I wasn't fully awake yet!